src/internal.c

```diff
@@ -408,6 +408,9 @@ const char* GetErrorString(int err)
         case WS_CERT_KEY_SIZE_E:
             return "key size too small error";
 
+        case WS_CTX_KEY_COUNT:
```

Comment:
Should we add an _E suffix to match the convention?

Reply:
Check